src/Relation/Binary/Construct/Closure/SymmetricTransitive.agda

```diff
@@ -0,0 +1,121 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Symmetric transitive closures of binary relations
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Relation.Binary.Construct.Closure.SymmetricTransitive where
+
+open import Level
+open import Function
+open import Relation.Binary
+
+private
+  variable
+    a ℓ ℓ′ : Level
+    A B    : Set a
+
+module _ {A : Set a} (_≤_ : Rel A ℓ) where
+  private
+    variable
+      x y z : A
```

Comment:
Do bad things happen when you lift these to the top-level `variable` declaration? You could get rid of some of the arguments to the equational reasoning combinators?

Reply:
Have we lifted the restriction on using `variable` only for levels and Sets?

Reply:
Ah no, we haven't! Good point :laughing: Yup, these should be removed!

Reply:
so what's the recommendation here?

Reply:
To remove the `x`,`y`,`z` variables entirely and add them manually :+1: